svnwiki-mode.el

```diff
@@ -1,11 +1,13 @@
-;;; svnwiki-mode.el --- Major mode for editing svnwiki markup
+;;; svnwiki-mode.el --- Major mode for editing svnwiki markup -*- lexical-binding: t -*-
```

Comment:
I only add that cookie when needed, but considering the move of Emacs core towards having it everywhere, I guess it's fine.

Reply:
Yep, small packages don't tend to require it but it's reasonable to propagate to help Emacs devs with future surveys of "how many packages do we still have that could be broken if we used lexical binding everywhere".